src/mono/browser/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -147,10 +147,23 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                     }
                 case "Debugger.scriptParsed":
                     {
-                        if (args["url"]?.ToString()?.Contains("/_framework/") == true) //is from dotnet runtime framework
```

Comment:
Keep the url string in a local to avoid the repeated access+tostring.

Reply:
done